src/commands/add.js

```diff
@@ -41,6 +41,10 @@ export function toAddOptions(
   Object.keys(DEPENDENCY_TYPE_FLAGS_MAP).forEach(depTypeFlag => {
     if (flags[depTypeFlag]) {
       type = DEPENDENCY_TYPE_FLAGS_MAP[depTypeFlag];
+      // check if value of dependency flag is a package name and then push to dependency arguments
+      if (typeof flags[depTypeFlag] === 'string') {
```

Comment:
kept it this way instead of creating const - so that if there are any updates in usage of meow this code can be removed with any further refactor

Reply:
Ah, nice solution.